src/buttons/join.ts

```diff
@@ -0,0 +1,38 @@
+import {
+  ActionRowBuilder,
+  ButtonInteraction,
+  EmbedBuilder,
+  SelectMenuOptionBuilder,
+  StringSelectMenuBuilder,
+} from "discord.js";
+import { languages } from "../types/help-roles";
+
+module.exports = {
+  name: "join",
+  async execute(interaction: ButtonInteraction) {
+    let cid = interaction.customId;
+    const userID = cid.split("-")[1];
+    let opts: SelectMenuOptionBuilder[] = [];
+    for (let lang of languages) {
+      opts.push(new SelectMenuOptionBuilder().setLabel(lang).setValue(lang));
```

Comment:
Use proper formatting.
```suggestion
      opts.push(new SelectMenuOptionBuilder()
         .setLabel(lang)
         .setValue(lang)
      );
```